README.md

```diff
@@ -81,15 +81,15 @@ Change to the branch with the name **develop**:
 
 and install the project dependencies:
 
-    $ yarn install
+    $ yarn install --network-timeout 1000000
```

Comment:
This looks very uncommon for an install step. I would just add one clarification sentence, that if somebody is experiencing issues with the install command he can add "--network-time...."

Reply:
I have updated the README to only suggest trying that command if they experience a problem with the first one.